### ISSUE TITLE
How to know the peaks called after clipper

### ISSUE THREAD

**Issue by ELMMA-FUNNY** [YeoLab/clipper#111]
Hello,after I used the clipper,I found the three results:
  4008101 Sep 13 20:30 EXAMPLE_PE.rep2_clip.A03.r1.fq.genome-mappedSo.rmDupSo.merged.r2.peakClusters.bed* 
  54949513 Sep 13 20:30 EXAMPLE_PE.rep2_clip.A03.r1.fq.genome-mappedSo.rmDupSo.merged.r2.peakClusters.bed.pickle*
  13 Sep 13 20:30 EXAMPLE_PE.rep2_clip.A03.r1.fq.genome-mappedSo.rmDupSo.merged.r2.peakClusters.bed.tsv*

I want to konw how many peaks I called 

and there is another question:
When i less EXAMPLE_PE.rep2_clip.A03.r1.fq.genome-mappedSo.rmDupSo.merged.r2.peakClusters.bed.tsv
its result is:
Nothing Here

Am I doing perfect in clipper?
I used clipper in docker:brianyee/clipper:Osativa7_branch

**Comment by byee4** [comment]
`wc -l <my_peaks.bed>`

What's your command? That branch should be valid/working

**Comment by ELMMA-FUNNY** [comment]
My command is:
make the container:
docker run -itd -v /home/lulu/chenhui_project/eclip/changxin_lab:/home/lulu/chenhui_project/eclip/changxin_labbrianyee/clipper:Osativa7_branch /bin/bash
and then start and attach the docker brianyee/clipper:Osativa7_branch

clipper \
--species hg19 \
--bam EXAMPLE_PE.rep2_clip.A03.r1.fq.genome-mappedSo.rmDupSo.merged.r2.bam \
--save-pickle \
--outfile EXAMPLE_PE.rep2_clip.A03.r1.fq.genomemappedSo.rmDupSo.merged.r2.peakClusters.bed

And after I wc -l <my_peaks.bed>
The results are:
45369 EXAMPLE_PE.rep2_clip.A03.r1.fq.genome-mappedSo.rmDupSo.merged.r2.peakClusters.bed
23652 INPUT_PE.rep2_clip.NIL.r1.fq.genome-mappedSo.rmDupSo.merged.r2.peakClusters.bed

**Comment by byee4** [comment]
Yup that appears right, however are you using the [example data](https://s3-us-west-1.amazonaws.com/external-collaborator-data/reference-data/204_01_RBFOX2.tar.gz) for paired end? If so, the numbers appear off from what we have (should be closer to ~150k peak clusters, which are then further normalized with the script below). 

Let's check to make sure you're using the latest script, as the one in gscripts is deprecated and is no longer being updated:
https://raw.githubusercontent.com/YeoLab/eclip/master/bin/overlap_peakfi_with_bam.pl
